packages/wallets/src/features/cashier/modules/WithdrawalCrypto/utils/withdrawalCryptoValidators.ts

```diff
@@ -0,0 +1,82 @@
+import { TWithdrawalCryptoContext } from '../provider';
+
+const helperMessageMapper = {
```

Comment:
@george-usynin-deriv PR had similiar patterns but with the <Trans component, might be cool to get in sync and use the same patterns everywhere (both seem equally valid, although personally I do not like "mapper" as concept as all the mappers eventually tends to grow and become just a bags for anything which does not easily fit anywhere else)

https://github.com/binary-com/deriv-app/pull/12378